src/preFilter.ts

```diff
@@ -53,6 +62,20 @@ export function filterByContent(searchee: Searchee): boolean {
 	return true;
 }
 
+export function releaseInBlockList(
+	searchee: Searchee,
+	blockList: string[]
+): boolean {
+	return blockList.some((str) => {
+		return (
+			searchee.name.includes(str) ||
+			(str.length === 40 &&
+				searchee.infoHash &&
+				str === searchee.infoHash)
```

Comment:
we can probably achieve the same thing in less comparisons with just `str === searchee.infoHash`
```suggestion
				str === searchee.infoHash
```